chapter2/tests/t11/nginx.conf

```diff
@@ -0,0 +1,68 @@
+user nginx nginx;
+worker_processes 1;
+pid /run/nginx.pid;
+
+events {
+	worker_connections 768;
+}
+
+http {
+
+	server {
+		server_name _;
+		listen 443 ssl;
+		ssl_certificate /tmp/certs/cert.pem;
+		ssl_certificate_key /tmp/certs/key.pem;
+
+		location / {
+			fastcgi_split_path_info ^(.+\.php)(.*)$;
+		}
+
+		location /reports/fast/ {
+			root /tmp/camp/reports/fast;
+			fastcgi_pass php-fpm;
+			try_files /fast.php =404;
+			include fastcgi_params;
+		}
+
+		location /reports/slow/ {
+			root /tmp/camp/reports/slow;
+			fastcgi_pass php-fpm-slow;
+			fastcgi_read_timeout 605;
+			try_files /slow.php =404;
+			include fastcgi_params;
+		}
+
+	}
+
+
+	upstream php-fpm-slow {
+		server unix:/run/php/php8.1-fpm-slow.sock fail_timeout=0;
+	}
+
+
+	upstream php-fpm {
+		server unix:/run/php/php8.1-fpm.sock;
+	}
+
+
+	sendfile on;
+	tcp_nopush on;
+	types_hash_max_size 2048;
+
+
+	include /etc/nginx/mime.types;
+	default_type application/octet-stream;
+
+
+	ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3; # Dropping SSLv3, ref: POODLE
+	ssl_prefer_server_ciphers on;
+
+
+	error_log /dev/stderr;
+	access_log /dev/stdout;
+
+
+	gzip on;
```

Comment:
that should go on top of the file. Your servers/locations should be at the bottom

Reply:
> that should go on top of the file. Your servers/locations should be at the bottom

@dmitry-mightydevops 
Done